plugin/src/main/java/com/h3xstream/findsecbugs/crypto/WeakTLSDetector.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Find Security Bugs
+ * Copyright (c) Philippe Arteau, All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3.0 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library.
+ */
+package com.h3xstream.findsecbugs.crypto;
+
+import edu.umd.cs.findbugs.BugInstance;
+import edu.umd.cs.findbugs.BugReporter;
+import edu.umd.cs.findbugs.OpcodeStack;
+import edu.umd.cs.findbugs.Priorities;
+import edu.umd.cs.findbugs.bcel.OpcodeStackDetector;
+import org.apache.bcel.Constants;
+
+public class WeakTLSDetector extends OpcodeStackDetector {
```

Comment:
I'm working on the test case for this the ApacheHttpClient library needs to be added as a test dependency - https://mvnrepository.com/artifact/org.apache.httpcomponents/httpclient.

I added it to plugin/test-dependencies.xml together with its own 3 dependencies, but it's not working. This is my first time working with Maven so I could use your help here.